packages/ccss/ccss.1.6/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "Dario Teixeira <dario.teixeira@nleyten.com>"
+authors: ["Dario Teixeira <dario.teixeira@nleyten.com>"]
+homepage: "http://ccss.forge.ocamlcore.org/"
+bug-reports: "https://github.com/darioteixeira/ccss/issues"
+dev-repo: "https://github.com/darioteixeira/ccss.git"
+license: "GPL-2.0"
+available: [ocaml-version >= "4.02.0"]
+build: [
+    ["./configure" "--prefix" prefix]
+    [make]
+]
+install: [[make "install"]]
+remove: [["rm" "%{bin}%/ccss"]]
+depends: [
+  "alcotest" {test}
+  "batteries" {>= "2"}
+  "menhir"
+  "ocamlbuild" {build}
+  "ocamlfind"
```

Comment:
@darioteixeira You have nothing to do, this will be automatically performed by the next release of `oasis2opam`.